vertical-pod-autoscaler/e2e/v1/admission_controller.go

```diff
@@ -157,7 +156,7 @@ var _ = AdmissionControllerE2eDescribe("Admission-controller", func() {
 			apiv1.ResourceCPU:    ParseQuantityOrDie("100m"),
 			apiv1.ResourceMemory: ParseQuantityOrDie("100Mi"),
 		}
-		klog.Infof("d: %+v", d)
+		framework.Logf("Created Hamster deployment %v", d)
```

Comment:
```suggestion
		framework.Logf("Created hamster deployment %v", d)
```

Just to be consistent with the other log messages above